base/leaky_bucket.go

```diff
@@ -283,7 +283,12 @@ func (b *LeakyBucket) UUID() (string, error) {
 		return b.bucket.UUID()
 }
 
-
+func (b *LeakyBucket) CloseAndDelete() error {
+	if bucket, ok := b.bucket.(sgbucket.DeleteableBucket); ok {
+		return bucket.CloseAndDelete()
+	}
+	return nil
+}
```

Comment:
Ah - didn't notice that the unit tests are always running as walrus wrapped w/ LeakyBucket (for walrus DCP emulation) after your changes to bucket.go.  Makes sense now - thanks.